wavedump/src/analyze-waveforms

```diff
@@ -328,11 +332,12 @@ if __name__ == '__main__':
     parser.add_argument('--chunks', default=10000, type=int, help='number of waveforms to process at a time')
     parser.add_argument('-t', '--integration-time', default=300, type=float, help='SPE integration length in nanoseconds.')
     parser.add_argument('-s', '--start-time',  default=50, type=float, help='start time of the SPE integration in nanoseconds.')
-    parser.add_argument('--active', default=None, help='Only take data from a single channel. If not specified, all channels are analyzed.')
+    parser.add_argument('--active', default=None, help='Channels to analyze data. For example "--active 123" will analyze data from ch1, ch2, and ch3. Leaving this argument blank analyzes all channels.')
     parser.add_argument('--integration-method', type=int, default=1, help='Select a method of integration. Methods described in __main__')
     parser.add_argument("--print-pdfs", default=None, type=str, help="Folder to save pdfs in.")
     parser.add_argument('-u','--upload', default=False, action='store_true', help='upload results to the database')
     parser.add_argument('-i','--institution', default=None, type=Institution, choices=list(Institution), help='name of institution')
+    parser.add_argument('--crosstalk', default=None, help='Channel numbers for crosstalk comparison. `01` means select 511 events only from ch0, and analyze ch1/ch0.')
```

Comment:
I think it's probably a lot more work to do, so we can chat about whether we want to put this off for another pull request, but eventually I think it would be nice to just automatically compute the crosstalk for every possible combination of channels and upload that information to the database.